src/components/ManifestDetailSidePanel/__tests__/ManifestDetailSidePanel.test.tsx

```diff
@@ -65,7 +81,40 @@ describe('Manifest Detail Side Panel', () => {
       }
     }));
 
-    const { container } = render(<ManifestDetailSidePanel {...props} />);
+    const { container } = render(
+      <QueryClientProvider client={queryClient}>
+        <ManifestDetailSidePanel {...props} />
+      </QueryClientProvider>
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it("renders successfully with data, but showing 'administratively disabled' for SCA status when user is not SCA capable", () => {
```

Comment:
I'd omit the first clause here about rendering successfully with data to shorten this up.  The SCA status is what we care about here.